codegen/templates/keyword.j2

```diff
@@ -153,4 +153,4 @@ class {{keyword_data.classname}}(KeywordBase):
 {% with links = keyword_data.links %}
 {% include 'keyword/links.j2' %}
 {% endwith %}
-{% include 'keyword/alias.j2' %}
+{% include 'keyword/alias.j2' %}
```

Comment:
Won't this newline here break the codegen check unless all the files are regenerated?